sympy/stats/crv_types.py

```diff
@@ -910,7 +910,7 @@ def _moment_generating_function(self, t):
 
 def Exponential(name, rate):
     r"""
-    Create a continuous random variable with an Exponential distribution.
+    Creates a Continuous Random Variable with Exponential distribution.
```

Comment:
I don't understand why you want to change this sentence. If you use `creates` you should add the subject first, e.g. _This function creates ..._. It makes sense to start the sentence with _Create_ but not with _Creates_.

Reply:
I would leave these sentences as they are.